deploy.sh

```diff
@@ -33,8 +33,8 @@ else
     export MAX_REPLICAS="2"
 
     export MEMBER_REFRESH_HOUR="17" # notprod turns off earlier than prod
-    
     export IS_PROD=""
+    export DOWNSCALER_UPTIME_ANNOTATION_VALUE="Mon-Fri 07:55-18:00 Europe/London"
```

Comment:
Consider calling it what it is:
```suggestion
    export UPTIME_PERIOD="Mon-Fri 07:55-18:00 Europe/London"
```